gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/EnvironmentService.java

```diff
@@ -43,4 +44,6 @@ public interface EnvironmentService {
     EnvironmentEntity findByCockpitId(String cockpitId);
 
     EnvironmentEntity getDefaultOrInitialize();
+
+    Set<EnvironmentEntity> findAllOrInitialize();
```

Comment:
Can `initialize()` be removed from the interface and be converted into a private method ?

Reply:
hmm, it is used by the DefaultEnvironmentUpgrader so I would say no